docs/remote.md

```diff
@@ -241,7 +241,7 @@ where `NetworkID` and `EndpointID` have meanings as above. The success response
 
 ### DiscoverNew Notification
 
-libnetwork listens to inbuilt docker discovery notifications and passes it along to the interested drivers. 
+Libnetwork listens to inbuilt docker discovery notifications and passes it along to the interested drivers. 
```

Comment:
I see "LibNetwork", "libnetwork" (which you are removing), "Libnetwork" (which you are adding) in this document.
Maybe we should just choose one and make it consistent.